components/Search/Search.jsx

```diff
@@ -67,10 +67,11 @@ const Search = ({
     query,
     hasQuery || showOnlyMyResults
   );
-  const results = data && {
-    records: data?.reduce((acc, d) => [...acc, ...getRecords(d)], []),
-    nextCursor: data[data.length - 1].nextCursor,
-  };
+  const results = data &&
+    data != 1 && {
```

Comment:
Probably best to use strict (in)equality: `data !== 1`. Same in your change further down too

Reply:
Done ;)